From f181899034beb4311c8e63056ce73d00df6981c8 Mon Sep 17 00:00:00 2001
From: Markus Volk <f_l_k@t-online.de>
Date: Sun, 30 Jun 2024 15:10:05 +0200
Subject: [PATCH] ScreencopyShared.hpp: drop #define
 SPA_VERSION_POD_BUILDER_CALLBACKS

This is defined in pipewire

Fixes build with pipewire 1.2.0
| ../git/src/portals/../core/../portals/../shared/ScreencopyShared.hpp:9:43: error: expected primary-expression before '.' token
|     9 | #define SPA_VERSION_POD_BUILDER_CALLBACKS .version = 0

Signed-off-by: Markus Volk <f_l_k@t-online.de>
---
 src/shared/ScreencopyShared.hpp | 4 +---
 1 file changed, 0 insertions(+), 2 deletions(-)

diff --git a/src/shared/ScreencopyShared.hpp b/src/shared/ScreencopyShared.hpp
index d626da6..e3b0a94 100644
--- a/src/shared/ScreencopyShared.hpp
+++ b/src/shared/ScreencopyShared.hpp
@@ -5,8 +5,6 @@
 extern "C" {
 #include <spa/pod/builder.h>
 
-#undef SPA_VERSION_POD_BUILDER_CALLBACKS
-#define SPA_VERSION_POD_BUILDER_CALLBACKS .version = 0
 #include <spa/buffer/meta.h>
 #include <spa/utils/result.h>
 #include <spa/param/props.h>
-- 
2.45.1

